src/server/session.rs

```diff
@@ -65,6 +68,8 @@ where
             data_cmd_rx: None,
             data_abort_tx: None,
             data_abort_rx: None,
+            internal_msg_tx: None,
```

Comment:
We may want to come up with a naming scheme for all our channels to make things clear. I know you didn't come op with internal message. Perhaps `data_cmd_tx` is not a bad example i.e. it uses scheme `{target_loop}_{msg_type}_{rx/tx}`.

Reply:
yes... i like your suggestion